pureStorageDriver/README.md

```diff
@@ -17,7 +17,7 @@ This helm chart installs the Pure Service Orchestrator CSI plugin on a Kubernete
   - Ubuntu 18.04
 - #### Environments Supported*:
   - Kubernetes 1.13+
-    - NOTE: for Kubernetes 1.17, there is an [issue](https://github.com/kubernetes/kubernetes/issues/87852) using vxlan with Flannel or Calico 
+    - NOTE: for Kubernetes 1.17, there is an [issue](https://github.com/kubernetes/kubernetes/issues/87852) using vxlan with Flannel or Calico, the issue has been fixed after 18.5
```

Comment:
1.18.5

Reply:
it's fixed in 1.18.6? or 1.18.5, it's a little bit confusing.

Reply:
I know, the change log says 1.18.6 but I tested 1.18.5 and it was already working, and looks like the actual PR was merged before 18.5 release